### PR TITLE
#101 - Create Danger Zone component

### LINKED ISSUE #101: Create Danger Zone component
1SP
image

### PR DESCRIPTION
**Pull request by Koushith**
## Description

 - [x] Adds Danger zone component.

## Screenshot
image

## Platform tested
- [x] Large
- [x] Tablet
- [x] Mobile

### CODE REVIEW

File: src/screens/safe-overview/components/danger-zone/danger-zone.component.styles.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { Button } from 'components/primitive';
+
+export const StyledRow = styled.span`
```
Comment (koshikraj) [inline src/screens/safe-overview/components/danger-zone/danger-zone.component.styles.ts]: Why are we not using `Box` here?

File: src/screens/safe-overview/components/danger-zone/danger-zone.component.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Box, Text } from 'components/primitive';
+import { StyledRow, DangerZoneButton } from './danger-zone.component.styles';
+
+export const DangerZone = () => {
+  return (
+    <Box color='white' padding={4} width={43.3}>
+      <Text variant='contentHeader' tx='safeOverViewPage.dangerZone' />
+      <Box marginTop={2.4}>
+        <StyledRow>
```
Comment (koshikraj) [inline src/screens/safe-overview/components/danger-zone/danger-zone.component.tsx]: Why are we not using `Box` with row property ?

**Review by koshikraj — CHANGES_REQUESTED** [review]

File: src/screens/safe-overview/components/danger-zone/danger-zone.component.styles.ts (hunk 1 above)
Comment (Koushith) [reply src/screens/safe-overview/components/danger-zone/danger-zone.component.styles.ts]: my bad- changes were in the staging area. pushed the latest code.

File: src/screens/safe-overview/components/danger-zone/danger-zone.component.tsx (hunk 2 above)
Comment (Koushith) [reply src/screens/safe-overview/components/danger-zone/danger-zone.component.tsx]: changed in the latest commit.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**